### PR TITLE
add LoRA inference notebook

### PR DESCRIPTION
**Pull request by zainhas**

### CODE REVIEW

File: .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+     "text": [
+      "{'id': '8f2cb236c80ea20e-YYZ', 'object': 'text.completion', 'created': 1734331375, 'model': 'zainhas/Meta-Llama-3.1-8B-Instruct-Reference-my-demo-finetune-4224205a', 'prompt': [], 'choices': [{'text': \"Here's a debate on the pros and cons of AI:\\n\\n**Moderator:** Welcome to today's debate on the pros and cons of AI. We have two debaters, Alex and Ben, who will present their arguments on the topic. Alex will argue in favor of AI, while Ben will argue against it. Let's begin with opening statements.\\n\\n**Alex (In Favor of AI):** Thank you, Moderator. AI has revolutionized the way we live and work. It has improved efficiency, productivity, and accuracy in various industries, such as healthcare, finance, and transportation. AI-powered systems can analyze vast amounts of data, identify\", 'finish_reason': 'length', 'seed': 5626645655383684000, 'logprobs': None, 'index': 0}], 'usage': {'prompt_tokens': 18, 'completion_tokens': 128, 'total_tokens': 146, 'cache_hit_rate': 0}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "\n",
+    "api_key = \"11d802b7e0038ef5c5cb7768d5e62e8416666a6e998361bba7a4db3031762f1c\"\n",
```
Comment (artek0chumak) [inline .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb:54]: Are you sure you want to have your API key here presented?

File: .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb
Comment (artek0chumak) [inline .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb:1]: Why not to remove this whole folder? Add it to .gitignore

**Review by artek0chumak — CHANGES_REQUESTED** [review]

File: .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb (hunk 1 above)
Comment (zainhas) [reply .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb]: was mistake - I have regenerated a new key

File: .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb
Comment (zainhas) [reply .ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb:1]: done

**Review by artek0chumak — APPROVED** [review]

**Pull request merged**